src/InstallationValidator.Core/Reporting/OutputComparisonResultTeXBuilder.cs

```diff
@@ -27,7 +26,6 @@ public override void Build(OutputComparisonResult outputToReport, OSPSuiteTracke
          {
             new SubParagraph($"{Assets.Reporting.OutputPath}: {outputToReport.Path}"),
             ValidationMessageFor(outputToReport),
-            new LineBreak(),
```

Comment:
LineBreak is added by the ValidationMessageFor. Thaty way, we do not create an empty line break when validation message was empty (creating an error on latex